lib/tvseries/scrapers/legends_of_tomorrow.rb

```diff
@@ -0,0 +1,52 @@
+module TVSeries
+  module Scrapers
+    class LegendsOfTomorrow < Base
+      WIKI_LINK      = 'https://en.wikipedia.org/wiki/Legends_of_Tomorrow'.freeze
+      IMDB_LINK      = 'http://www.imdb.com/title/tt4532368/'.freeze
+      SCRAPE_LINK    = 'https://en.wikipedia.org/wiki/List_of_Legends_of_Tomorrow_episodes'.freeze
+      JSON_FILE_PATH = File.join(TVSeries::SERIES_JSON_PATH, 'legends_of_tomorrow.json').freeze
+      SHORT_NAME     = 'lot'.freeze
```

Comment:
For all scraper classes, store just the `series_name` (short name) and access other details from the `base.json`.